src/Compilers/Test/Core/CollectionBuilderAttribute.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if false
```

Comment:
😕 Why was this needed?

Reply:
The type was added at the core layer.  So this one became unneeded